src/engine/sys/con_curses.cpp

```diff
@@ -184,7 +184,7 @@ static void CON_UpdateClock()
 	qtime_t realtime;
 	Com_Time( &realtime );
 	werase( clockwin );
-	CON_ColorPrint( clockwin, va( "^0[^3%02d%c%02d^0]^7 ", realtime.tm_hour, ( realtime.tm_sec & 1 ) ? ':' : ' ', realtime.tm_min ), true );
```

Comment:
I assumed this to be followed by the cvar `con_prompt`, but apparently in this instance it uses a hard coded value instead.
Perhaps it might be better leaving this in, and considering supporting the already existing cvar later?

Reply:
Supporting `con_prompt` seems fairly pointless to me, but either way it doesn't makes sense to keep a color code at the end of a line.